include/mbedtls/lms.h

```diff
@@ -86,14 +90,16 @@ extern "C" {
  */
 typedef enum {
     MBEDTLS_LMS_SHA256_M32_H10 = 0x6,
+    MBEDTLS_LMS_SHA256_M24_H20 = 0xd,
```

Comment:
We'll need a [changelog entry](https://github.com/Mbed-TLS/mbedtls/blob/development/ChangeLog.d/00README.md) for the new feature.

Reply:
Do I just add this to the LMS.txt in the changelog folder?

Reply:
Please make a separate file, because there's a good chance that we'll make a release before this pull request can be reviewed, and `ChangeLog.d/*` is consolidated into `ChangeLog` at release time.

Reply:
Done. PTAL